zerver/tests/test_openapi.py

```diff
@@ -275,6 +275,8 @@ class OpenAPIArgumentsTest(ZulipTestCase):
         "/rest-error-handling",
         # Zulip outgoing webhook payload
         "/zulip-outgoing-webhook",
+        # Api for previewable url
+        "/previewable",
```

Comment:
We're going to want to document this in the OpenAPI spec eventually, since we'd like any client to be able to use it.

Reply:
Added it here because checks were failing, I will add an issue or put a PR after this PR closes.